cirq-core/cirq/protocols/json_test_data/SycamoreTargetGateset.repr

```diff
@@ -0,0 +1 @@
+cirq_google.SycamoreTargetGateset(atol=1e-08, tabulation=None)
```

Comment:
The repr json combo might need to go in the cirq_google json_test_data directory ?

Reply:
Oops my bad, thanks for the catch!